com.ca.lsp.cobol/lsp-core-cobol-parser/src/main/java/com/ca/lsp/core/cobol/preprocessor/sub/document/impl/CobolSemanticParserListenerImpl.java

```diff
@@ -132,12 +137,15 @@ public void exitCopyStatement(@Nonnull CopyStatementContext ctx) {
     /*
      * define the copy book
      */
-    CopySourceContext copySource = ctx.copySource();
-    String copybookName = retrieveCopybookName(copySource);
-    Position position = retrievePosition(copySource);
-    defineCopybook(copybookName, position);
-    checkCopybookNameLength(copybookName, position);
-    this.preprocessorCleanerService.excludeStatementFromText(ctx, COMMENT_TAG, tokens);
+
+    if (copybookScanAnalysis.equalsIgnoreCase("ENABLED")) {
+      CopySourceContext copySource = ctx.copySource();
+      String copybookName = retrieveCopybookName(copySource);
+      Position position = retrievePosition(copySource);
+      defineCopybook(copybookName, position);
+      checkCopybookNameLength(copybookName, position);
```

Comment:
I'm not sure if we need this check while analyzing the extended document. In fact, it checks the semantics, but I don't know if it will be useful for the user at that stage

Reply:
well.. the purpose here is to don't throw syntax errors for copybooks that are not found